packages/cli/src/commands/test.js

```diff
@@ -41,18 +42,12 @@ export const builder = (yargs) => {
       type: 'boolean',
       default: true,
     })
-    .option('collectCoverage', {
+    .option('collect-coverage', {
```

Comment:
We use both styles in our CLI; e.g., [`yarn rw deploy`](https://redwoodjs.com/docs/cli-commands#deploy-netlify) takes a `--data-migrate` option, not a `--dataMigrate` option. In `yarn rw generate layout` I added [`--skipLink`](https://redwoodjs.com/docs/cli-commands#generate-layout). And `yarn rw serve api` takes [`--apiRootPath`](https://redwoodjs.com/docs/cli-commands#serve-api). What should we do?

Reply:
Both are actually supported @jtoar. If you pass `--colectCoverage` it'll imply `--collect-coverage`. No breaky! Using the dash case, when we define it makes it easier to filter out this param though, before we pass other args to jest.

Reply:
Ah gotcha!